corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -257,7 +257,7 @@ def can_use_restore_as(request):
     return (
         request.couch_user.can_edit_commcare_users() and
         has_privilege(request, privileges.DATA_CLEANUP) and
```

Comment:
realizing that `DATA_CLEANUP` is only for pro plan right? so this pretty much means the `LOGIN_AS` priv isn't helpful. should we remove this as well?